app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
+  skip_before_action :require_login, only: [:new, :create]
+
+  def new
+  end
+
+  def create
+    auth_hash = request.env['omniauth.auth']
+    user = User.find_or_create_from_omniauth(auth_hash)
+    # binding.pry
```

Comment:
It's probably a good idea to get in the habit of making sure you don't commit comments for yourself, bindings, etc. In internships/jobs people care a lot about this!